components/dashboard/src/Menu.tsx

```diff
@@ -100,6 +98,7 @@ export default function Menu() {
 
     // Hide most of the top menu when in a full-page form.
     const isMinimalUI = ["/new", "/teams/new", "/open"].includes(location.pathname);
+    const isWorkspacesUI = ["/workspaces"].includes(location.pathname);
```

Comment:
**thought**: The idea here is that for the workspaces' page we'd like to remove the tabs but also keep the navigation up there.